supply-chain-security/compromises/2007/wordpress.md

```diff
@@ -0,0 +1,11 @@
+# Backdoor in WordPress as a result of server compromise
+
+A cracker gained user-level access to a server behind wordpress.org, added a backdoor in the code.
+
+## Impact
+
+The backdoor made it into a WordPress release but a new one was released shortly.
+
+## Type of compromise
+
+Infrastructure compromise with source code access, no signatures/authenticity.
```

Comment:
Could you add `(Publishing Infrastructure)` and `(Source Code)` here to match [definitions](https://github.com/cncf/sig-security/blob/master/supply-chain-security/compromises/compromise-definitions.md).  Based on the details of the compromise it appears to be publishing infrastructure.